pkgs/development/lisp-modules/quicklisp-to-nix-output/chunga.nix

```diff
@@ -3,7 +3,7 @@ args @ { fetchurl, ... }:
   baseName = ''chunga'';
   version = ''20180131-git'';
 
-  description = '''';
+  description = ''System lacks description'';
```

Comment:
I'm guessing this was generated. But maybe we could back out this and similar changes so the diff is smaller.

Reply:
Since this has been generated, I would prefer to not edit it manually.